docs/guides/extension_points.rst

```diff
@@ -72,6 +75,7 @@ For a more detailed comparison of content integration options, see `Options for
 .. _Options for Extending the edX Platform: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/extending_platform/extending.html
 .. _custom JavaScript application: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/extending_platform/javascript.html
 .. _external grader documentation: https://edx.readthedocs.io/projects/open-edx-ca/en/latest/exercises_tools/external_graders.html
+.. _the additional configuration settings for TinyMCE: https://github.com/edx/edx-platform/pull/25695
```

Comment:
To answer your other question, yes it would be better if this can link to a .rst documentation file somewhere, rather than a pull request.

Reply:
Yeah, I created a new directory in the `docs/guides` in order to add any guide close to the `extension_points` you mentioned.

Let me know if you think there's a better place for it, please.

This has been addressed in [`e6803fa - Updates the extension points and adds a guide on how to install custom tinymce plugins`](https://github.com/edx/edx-platform/pull/25695/commits/e6803fab65f0e35badf21addb214c46819b5fc91) 👍🏼